borg/archiver.py

```diff
@@ -1521,7 +1526,7 @@ def main():  # pragma: no cover
         msg = 'Local Exception.\n%s\n%s' % (traceback.format_exc(), sysinfo())
         exit_code = EXIT_ERROR
     except KeyboardInterrupt:
-        msg = 'Keyboard interrupt.\n%s\n%s' % (traceback.format_exc(), sysinfo())
+        msg = 'Keyboard interrupt.'
```

Comment:
Ok, that's a good reason to keep it for ^C. And a good reason to use another exception for SIGTERM :)